testsuite/features/secondary/proxy_container_retail_mass_import.feature

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2018-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped:
+# * if there is no proxy ($proxy is nil)
+# * if there is no private network ($private_net is nil)
+# * if there is no PXE boot minion ($pxeboot_mac is nil)
+
+@containerized_server
+@skip_if_github_validation
+@proxy
+@private_net
+@pxeboot_minion
+@scope_retail
+Feature: Mass import of Retail terminals behind a containerized proxy
```

Comment:
I wonder the reason behind using Retail instead of retail, but for instance, we recommend to use salt minions instead of Salt minions.

Reply:
According to our rules we use the capitalizations chosen by the owner of the product.

In this case the official version is "SUSE Manager for Retail" - complete wording.

Now, if we consider it's not a product, but terminals used in retail branch of economy, it's lower case...

Reply:
Keeping unchanged.